### PR TITLE
Qf 2008 provider view of employer requests

### PR DESCRIPTION
**Pull request by karen-hanna**

### CODE REVIEW

File: src/Shared/SFA.DAS.SharedOuterApi.UnitTests/InnerApi/Requests/WhenBuildingGetAggregatedEmployerRequestsRequest.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.ProviderRequestApprenticeTraining.InnerApi.Requests;
+
+namespace SFA.DAS.SharedOuterApi.UnitTests.InnerApi.Requests
+{
+    public class WhenBuildingGetAggregatedEmployerRequestsRequest
```
Comment (dashton82) [inline src/Shared/SFA.DAS.SharedOuterApi.UnitTests/InnerApi/Requests/WhenBuildingGetAggregatedEmployerRequestsRequest.cs:8]: This test should be in the SFA.DAS.ProviderRequestApprenticeTraining project

File: src/Shared/SFA.DAS.SharedOuterApi.UnitTests/SFA.DAS.SharedOuterApi.UnitTests.csproj [hunk 2]
```diff
@@ -24,6 +24,7 @@
     </ItemGroup>
 
     <ItemGroup>
+      <ProjectReference Include="..\..\ProviderRequestApprenticeTraining\SFA.DAS.ProviderRequestApprenticeTraining\SFA.DAS.ProviderRequestApprenticeTraining.csproj" />
```
Comment (dashton82) [inline src/Shared/SFA.DAS.SharedOuterApi.UnitTests/SFA.DAS.SharedOuterApi.UnitTests.csproj:27]: This reference shouldnt be here

File: src/ProviderRequestApprenticeTraining/SFA.DAS.ProviderRequestApprenticeTraining/Application/Queries/GetAggregatedEmployerRequests/GetAggregatedEmployerRequestsQueryHandler.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            _requestApprenticeshipTrainingApiClient = requestApprenticeTrainingApiClient;
+            _logger = logger;
+        }
+
+        public async Task<GetAggregatedEmployerRequestsResult> Handle(GetAggregatedEmployerRequestsQuery request, CancellationToken cancellationToken)
+        {
+            var aggregatedEmployerRequests = await _requestApprenticeshipTrainingApiClient.
+                Get<List<GetAggregatedEmployerRequestsResponse>>(new GetAggregatedEmployerRequestsRequest());
+
+            _logger.LogDebug($"End GetAggregatedEmployerRequestsHandler");
```
Comment (dashton82) [inline src/ProviderRequestApprenticeTraining/SFA.DAS.ProviderRequestApprenticeTraining/Application/Queries/GetAggregatedEmployerRequests/GetAggregatedEmployerRequestsQueryHandler.cs:32]: can remove debug comments - less noise 😁

File: src/ProviderRequestApprenticeTraining/SFA.DAS.ProviderRequestApprenticeTraining.Api/Startup.cs [hunk 4]
```diff
@@ -56,6 +55,7 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddMediatR(c => c.RegisterServicesFromAssembly(typeof(GetEmployerRequestQuery).Assembly));
             services.AddMediatR(c => c.RegisterServicesFromAssembly(typeof(GetRoatpV2ProviderQuery).Assembly));
+            services.AddMediatR(c => c.RegisterServicesFromAssembly(typeof(GetAggregatedEmployerRequestsQuery).Assembly));
```
Comment (dashton82) [inline src/ProviderRequestApprenticeTraining/SFA.DAS.ProviderRequestApprenticeTraining.Api/Startup.cs:58]: should only need one of these lines here.

File: src/ApprenticeFeedback/SFA.DAS.ApprenticeFeedback/Application/Queries/GetAggregatedEmployerRequests/GetAggregatedEmployerRequestsQueryHandler.cs [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+            _apprenticeFeedbackApiClient = apprenticeFeedbackApiClient;
+            _logger = logger;
+        }
+
+        public async Task<GetAggregatedEmployerRequestsResult> Handle(GetAggregatedEmployerRequestsQuery request, CancellationToken cancellationToken)
+        {
+            var result = await _apprenticeFeedbackApiClient.
+                Get<GetAggregatedEmployerRequestsResult>(new GetAggregatedEmployerRequestsRequest());
+
+            _logger.LogDebug($"End Get Aggregated Employer Requests Handler");
```
Comment (dashton82) [inline src/ApprenticeFeedback/SFA.DAS.ApprenticeFeedback/Application/Queries/GetAggregatedEmployerRequests/GetAggregatedEmployerRequestsQueryHandler.cs:29]: can remove this debug log

**Review by dashton82 — CHANGES_REQUESTED** [review]

File: src/Shared/SFA.DAS.SharedOuterApi.UnitTests/InnerApi/Requests/WhenBuildingGetAggregatedEmployerRequestsRequest.cs (hunk 1 above)
Comment (karen-hanna) [reply src/Shared/SFA.DAS.SharedOuterApi.UnitTests/InnerApi/Requests/WhenBuildingGetAggregatedEmployerRequestsRequest.cs:8]: I've moved it to the correct project and also added a test for the corresponding response

File: src/Shared/SFA.DAS.SharedOuterApi.UnitTests/SFA.DAS.SharedOuterApi.UnitTests.csproj (hunk 2 above)
Comment (karen-hanna) [reply src/Shared/SFA.DAS.SharedOuterApi.UnitTests/SFA.DAS.SharedOuterApi.UnitTests.csproj:27]: Reference removed

File: src/ProviderRequestApprenticeTraining/SFA.DAS.ProviderRequestApprenticeTraining/Application/Queries/GetAggregatedEmployerRequests/GetAggregatedEmployerRequestsQueryHandler.cs (hunk 3 above)
Comment (karen-hanna) [reply src/ProviderRequestApprenticeTraining/SFA.DAS.ProviderRequestApprenticeTraining/Application/Queries/GetAggregatedEmployerRequests/GetAggregatedEmployerRequestsQueryHandler.cs:32]: Removed

File: src/ProviderRequestApprenticeTraining/SFA.DAS.ProviderRequestApprenticeTraining.Api/Startup.cs (hunk 4 above)
Comment (karen-hanna) [reply src/ProviderRequestApprenticeTraining/SFA.DAS.ProviderRequestApprenticeTraining.Api/Startup.cs:58]: Removed

File: src/ApprenticeFeedback/SFA.DAS.ApprenticeFeedback/Application/Queries/GetAggregatedEmployerRequests/GetAggregatedEmployerRequestsQueryHandler.cs (hunk 5 above)
Comment (karen-hanna) [reply src/ApprenticeFeedback/SFA.DAS.ApprenticeFeedback/Application/Queries/GetAggregatedEmployerRequests/GetAggregatedEmployerRequestsQueryHandler.cs:29]: Removed

**Review by ChrisJWoodcock — CHANGES_REQUESTED** [review]
Are the files under the ApprenticeFeedback committed by mistake?

**Comment by karen-hanna** [comment]
Removed the files incorrectly added to ApprenticeFeedback

**Review by ChrisJWoodcock — DISMISSED** [review]

**Review by ChrisJWoodcock — APPROVED** [review]

**Review by ChrisJWoodcock — APPROVED** [review]

**Review by ChrisJWoodcock — APPROVED** [review]

**Pull request merged**